ai/executors/play_executor.py

```diff
@@ -76,8 +76,9 @@ def order_change_of_sta(self, cmd: STAChangeCommand):
             self._change_tactic(cmd)
 
     def _change_strategy(self, cmd: STAChangeCommand):
-        new_strategy = self.play_state.get_new_strategy(cmd.data["strategy"])
-        self.play_state.set_strategy(new_strategy)
+        new_strategy_class = self.play_state.get_new_strategy(cmd.data["strategy"])
```

Comment:
get/set is really C style. python should use property getter and setter:
```python
self.play_state.current_strategy = cmd.data['strategy']
```

The setter should do all the mechanic

Reply:
Yeah, it's really old code.